From 029d5c264bacbe03346b27f08f3066364fc83f44 Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Fri, 21 Oct 2022 11:21:00 +0000
Subject: [PATCH] Add mising texfat

---
 main.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/main.cpp b/main.cpp
index 3af0dcf..149dbaf 100644
--- a/main.cpp
+++ b/main.cpp
@@ -68,6 +68,7 @@ int main(int argc, char** argv) {
 
     LOG(DEBUG) << "Detected support for:"
                << (android::vold::IsFilesystemSupported("exfat") ? " exfat" : "")
+               << (android::vold::IsFilesystemSupported("exfat") ? " texfat" : "")
                << (android::vold::IsFilesystemSupported("ext4") ? " ext4" : "")
                << (android::vold::IsFilesystemSupported("f2fs") ? " f2fs" : "")
                << (android::vold::IsFilesystemSupported("ntfs") ? " ntfs" : "")
-- 
2.25.1

