From b271b0aaf881cb2a2269001747cd9a86d544178a Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Fri, 21 Oct 2022 11:41:47 +0000
Subject: [PATCH] Track changes for makefile

---
 base.mk | 19 +++++++++++++------
 1 file changed, 13 insertions(+), 6 deletions(-)

diff --git a/base.mk b/base.mk
index 2f35a80..1c91a8e 100644
--- a/base.mk
+++ b/base.mk
@@ -241,12 +241,11 @@ include build/make/target/product/gsi_release.mk
 PRODUCT_COPY_FILES += \
 	device/phh/treble/files/com.android.deskclock_whitelist.xml:system/etc/sysconfig/com.android.deskclock_whitelist.xml
 
-$(call inherit-product, vendor/arrow/config/common.mk)
-$(call inherit-product, vendor/arrow/config/BoardConfigSoong.mk)
-$(call inherit-product, vendor/arrow/config/BoardConfigArrow.mk)
-$(call inherit-product, device/arrow/sepolicy/common/sepolicy.mk)
-$(call inherit-product, vendor/arrow/config/bootanimation.mk)
--include vendor/arrow/build/core/config.mk
+$(call inherit-product, vendor/ancient/config/common.mk)
+$(call inherit-product, vendor/ancient/config/common_full_phone.mk)
+$(call inherit-product, vendor/ancient/config/BoardConfigSoong.mk)
+$(call inherit-product, device/ancient/sepolicy/common/sepolicy.mk)
+-include vendor/ancient/build/core/config.mk
 TARGET_NO_KERNEL_OVERRIDE := true
 TARGET_NO_KERNEL_IMAGE := true
 SELINUX_IGNORE_NEVERALLOWS := true
@@ -260,4 +259,12 @@ TARGET_FACE_UNLOCK_SUPPORTED := true
 PRODUCT_PACKAGES += \
     GCamGOPrebuilt
 
+#prebuilts
+PRODUCT_PACKAGES += \
+  OpenEUICC \
 
+#ANCIENT_GAPPS := true
+ANCIENT_OFFICIAL := true
+#FORCE_NEXUSLAUNCHER=true
+#ANCIENT_WEEABO=true
+WITH_ASSISTANT := true
-- 
2.25.1

