From 8a8d8b704ec4e5e086b78dd8179e8424bce77804 Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Fri, 21 Oct 2022 11:24:36 +0000
Subject: [PATCH] Remove adb-flag

---
 config/common.mk | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index 9daeea59..ab782e63 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -14,8 +14,8 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.usb.config=adb
 else
 # Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.usb.config=none
+#PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
+#PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.usb.config=none
 
 # Disable extra StrictMode features on all non-engineering builds
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.strictmode.disable=true
@@ -190,3 +190,5 @@ PRODUCT_PACKAGES += \
 endif
 
 -include $(WORKSPACE)/build_env/image-auto-bits.mk
+
+TARGET_NO_KERNEL_OVERRIDE := true
-- 
2.25.1

